README.md

```diff
@@ -66,3 +66,30 @@ work correctly.
    robots, press "SEND REPORTED" on the settings page. This will just update the
    reported state of the robots with the desired robot commands that are set in
    the game page after pressing the "Fight!" button.
+
+# How to play
+
+## Roles
+
+### Admin
+
+The admin has the responsability to assign robots to teams. Also, this role is
+able to set robots positions and rotation.
+
+#### Assign robots to teams
+
+Below the game map, there is a list of the detected robots. To assign a robot to
+the team, write the name of the team into the input field or select it directly
+from the teams options below the input field. After pick the robot team, the
+robot on the game map should turn the same color than the team.
+
+#### Positioning and rotation
```

Comment:
Add documentation to inform about new features and changes done.

This can be improve adding images for example, but I think we can invest more time on this later and continue focusing on having a functional game by the moment .